azure-mgmt-containerservice/src/test/java/com/microsoft/azure/management/containerservice/KubernetesClustersTests.java

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.management.containerservice;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.util.HashMap;
+import java.util.Properties;
+
+import com.google.common.base.Charsets;
+import com.google.common.io.Files;
+import com.microsoft.azure.management.resources.fluentcore.arm.Region;
+import com.microsoft.azure.management.resources.fluentcore.utils.SdkContext;
+import com.microsoft.rest.serializer.JacksonAdapter;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+
+public class KubernetesClustersTests extends ContainerServiceManagementTest {
+    private static final String sshKey = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCfSPC2K7LZcFKEO+/t3dzmQYtrJFZNxOsbVgOVKietqHyvmYGHEC0J2wPdAqQ/63g/hhAEFRoyehM+rbeDri4txB3YFfnOK58jqdkyXzupWqXzOrlKY4Wz9SKjjN765+dqUITjKRIaAip1Ri137szRg71WnrmdP3SphTRlCx1Bk2nXqWPsclbRDCiZeF8QOTi4JqbmJyK5+0UqhqYRduun8ylAwKKQJ1NJt85sYIHn9f1Rfr6Tq2zS0wZ7DHbZL+zB5rSlAr8QyUdg/GQD+cmSs6LvPJKL78d6hMGk84ARtFo4A79ovwX/Fj01znDQkU6nJildfkaolH2rWFG/qttD azjava@javalib.Com";
+
+    @Test
+    public void canCRUDKubernetesCluster() throws Exception {
+        String aksName = SdkContext.randomResourceName("aks", 15);
+        String dnsPrefix = SdkContext.randomResourceName("dns", 10);
+        String agentPoolName = SdkContext.randomResourceName("ap0", 10);
+        String servicePrincipalClientId = "spId";
+        String servicePrincipalSecret = "spSecret";
+
+        String envSecondaryServicePrincipal = System.getenv("AZURE_AUTH_LOCATION_2");
```

Comment:
document what this AZURE_AUTH_LOCATION_2 for.

Reply:
Done